packages/components/spinner/src/spinner.js

```diff
@@ -0,0 +1,47 @@
+import { TSElement, unsafeCSS, html, customElementDefineHelper } from '@tradeshift/elements';
+import '@tradeshift/elements.cover';
+import css from './spinner.css';
+import { types, sizes, colors } from './utils';
+
+customElementDefineHelper(
+	'ts-spinner',
+	class extends TSElement {
+		static get styles() {
+			return [TSElement.styles, unsafeCSS(css)];
+		}
+
+		static get properties() {
+			return {
+				color: { type: String, attribute: 'data-color' },
+				message: { type: String, attribute: 'data-message' },
+				size: { type: String, attribute: 'data-size' },
+				type: { type: String, attribute: 'data-type' },
+				visible: { type: Boolean, attribute: 'data-visible' }
+			};
+		}
+
+		constructor() {
+			super();
+			this.color = colors.BLUE;
+			this.message = '';
+			this.size = sizes.LARGE;
+			this.type = types.LIGHT;
+			this.visible = false;
+		}
+
+		render() {
+			return html`
+				${this.visible
+					? html`
+							<div class="spinner"></div>
+							${this.message
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Extract this nested ternary operation into an independent statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=javascript%3AS3358)

Reply:
I agree with this one as well ☝️ 
Maybe a separate getter?